doc/users/interactive.rst

```diff
@@ -146,45 +139,43 @@ Interactive mode controls:
 
 - whether created figures are automatically shown
 - whether changes to artists automatically trigger re-drawing existing figures
-- whether `.pyplot.show` returns immediately or after all of the
-  figures have been closed when given no arguments
-
+- when `.pyplot.show` exits: immediately, or after all of the figures have been closed if given no arguments
 
-If in interactive mode, then:
+Interactive mode:
 
-- newly created figures will be shown immediately
-- figures will automatically redraw on change
-- pyplot.show will return immediately by default
+- newly created figures will be displayed immediately
+- figures will automatically redraw when elements are changed
+- `pyplot.show` displays the figures and immediately returns to the prompt
 
-If not in interactive mode then:
+Not in interactive mode:
```

Comment:
```suggestion
If not in interactive mode:
```

Having "If"s here makes more sense to me in terms of parsing this line and the bullets as whole sentences, but not a big deal.